esx_service/vmdk_ops.py

```diff
@@ -854,11 +854,14 @@ def findDeviceByPath(vmdk_path, vm):
         # to match with the given volume name.
         # Filename format is as follows:
         #   "[<datastore name>] <parent-directory>/tenant/<vmdk-descriptor-name>"
-        logging.debug("d.backing.fileName %s", d.backing.fileName)
-        backing_disk = d.backing.fileName.split(" ")[1]
+        # Assume there is one space between these two parts "[<datastore name>]" and
+        # <parent-directory>/tenant/<vmdk-descriptor-name>
+        logging.debug("findDeviceByPath: d.backing.fileName %s", d.backing.fileName)
+        backing_disk = d.backing.fileName.rsplit(" ", 1)[1]
```

Comment:
I did some test.
Vsphere allows volume name including space since I can create a VM with name including space.

But if I create a volume using local driver, it does not allow volume name to include space, and I got the following error.